Mergin/validation.py

```diff
@@ -217,6 +219,18 @@ def check_db_schema(self):
                 if not has_change:
                     self.issues.append(SingleLayerWarning(lid, Warning.DATABASE_SCHEMA_CHANGE))
 
+    def check_field_names(self):
+        for lid, layer in self.layers.items():
+            if lid not in self.editable:
+                continue
+            dp = layer.dataProvider()
+            if dp.storageType() == "GPKG":
+                fields = layer.fields()
+                for f in fields:
+                    if '\n' in f.name():
```

Comment:
is it only forbidden character? Whatabout `{}[];?/\` are those ok to use?

Reply:
Added more characters